subprojects/build-cache/src/main/java/org/gradle/caching/internal/controller/BuildCacheCommandFactory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.caching.internal.controller;
+
+import com.google.common.collect.ImmutableSortedMap;
+import org.gradle.caching.BuildCacheKey;
+import org.gradle.caching.internal.CacheableEntity;
+import org.gradle.caching.internal.origin.OriginMetadata;
+import org.gradle.internal.fingerprint.CurrentFileCollectionFingerprint;
+import org.gradle.internal.snapshot.FileSystemSnapshot;
+
+import java.util.Map;
+
+public interface BuildCacheCommandFactory {
+    BuildCacheLoadCommand<LoadMetadata> createLoad(BuildCacheKey cacheKey, CacheableEntity entity);
```

Comment:
⭕️ This looks a little bit different on ower site, I need to figure out why.

Reply:
This is fixed on your side now, right?

Reply:
Yes, I'll merge the PR this morning.